shared/src/main/scala/tastyquery/ast/Types.scala

```diff
@@ -538,7 +543,7 @@ object Types {
         val computed = computeName
         myName = computed
         computed
-      else local
+      else local.asInstanceOf[ThisName]
```

Comment:
this one not strictly necessary but it seems to cause the debugger expression evaluator problems if its not there @adpi2 (for example breakpoint inside of `Types.ErasedTypeRef.preErase`)

Reply:
Add a comment here, so that we don't remove it later for cleaning up?

Reply:
The problem here is that the scalac options are not used by the expression evaluator.

I am currently working on it.